apps/ledger-live-desktop/src/renderer/styles/StyleProviderV3.tsx

```diff
@@ -41,8 +41,8 @@ const StyleProviderV3 = ({ children, selectedPalette }: Props) => {
   );
 };
 
-export const withV3StyleProvider = (Component: React.ComponentType<any>) => {
-  const WrappedComponent = (props: any) => {
+export const withV3StyleProvider = <T,>(Component: React.ComponentType<T>) => {
```

Comment:
Is it normal to have a comma right after the `T` here ?
or maybe it should be like `ThemedComponent` ?
```suggestion
export const withV3StyleProvider<T> = (Component: React.ComponentType<T>) => {
```

Reply:
yes, otherwise the .tsx parser goes nuts.